<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1905</width>
    <height>1204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWebEngineView" name="navegador" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>1081</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(129, 129, 129);</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="barra_progreso">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>1121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="img_fondo">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-5</y>
      <width>1911</width>
      <height>1171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>fondo.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_vel_h">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vel_h">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Velocidad Horizontal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vel_v">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Velocidad Vertical</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_vel_v">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_satelites">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_satelites">
    <property name="geometry">
     <rect>
      <x>698</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nº Satéllites</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_latitud">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>260</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Latitud</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_latitud">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_altitud">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>431</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_altitud">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>380</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Altitud</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hora">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Hora</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_hora">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_longitud">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_longitud">
    <property name="geometry">
     <rect>
      <x>718</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Longitud</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_movimiento">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_movimiento">
    <property name="geometry">
     <rect>
      <x>708</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Movimiento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_fecha">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.394737 rgba(104, 0, 0, 255), stop:0.721053 rgba(113, 0, 0, 255), stop:0.936842 rgba(255, 255, 255, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_hora_2">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Fecha</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_letras">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>1021</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Letras_geolocalizador_GPS.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_k1michel">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>1050</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo_obs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>img_fondo</zorder>
   <zorder>navegador</zorder>
   <zorder>barra_progreso</zorder>
   <zorder>in_vel_h</zorder>
   <zorder>label_vel_h</zorder>
   <zorder>label_vel_v</zorder>
   <zorder>in_vel_v</zorder>
   <zorder>in_satelites</zorder>
   <zorder>label_satelites</zorder>
   <zorder>label_latitud</zorder>
   <zorder>in_latitud</zorder>
   <zorder>in_altitud</zorder>
   <zorder>label_altitud</zorder>
   <zorder>label_hora</zorder>
   <zorder>in_hora</zorder>
   <zorder>in_longitud</zorder>
   <zorder>label_longitud</zorder>
   <zorder>in_movimiento</zorder>
   <zorder>label_movimiento</zorder>
   <zorder>in_fecha</zorder>
   <zorder>label_hora_2</zorder>
   <zorder>img_letras</zorder>
   <zorder>img_k1michel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1905</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
